Docs for https://github.com/mautic/mautic/pull/16123 (#714)
mautic/user-documentation: docs/components/forms.rst

  :alt: Screenshot showing domain blocking used in a Mautic Form

It's advised to provide a helpful message to display if the visitor tries to use an Email address from a blocked domain, to help them understand what the problem is.

.. vale off

Form data maintenance commands
******************************

.. vale on

> Mautic provides CLI commands to clean up orphan Form data. These commands are useful for database maintenance when Form submissions or Forms have been deleted but related data remains.

Reviewer: We need to change the passive to active voice.

```suggestion
Mautic provides CLI commands to clean up orphan Form data. These commands are useful for database maintenance when you delete Forms or Form submissions but related data remains.
```

Deleting orphan Form results tables
===================================

When you delete a Form in Mautic, the associated ``form_results_<form_id>_<form_alias>`` table may remain in the database. To remove these orphan tables:

.. code-block:: shell

   bin/console mautic:forms:delete-results-table

This command identifies Form results tables that no longer have an associated Form and removes them from the database. This helps reduce database size and keeps your database clean.

Deleting orphan Form submission records
=======================================

> When Form submissions are deleted, the corresponding records in the Form results table may not always be removed. To clean up these orphan records:

Reviewer: Use active instead of passive voice.

```suggestion
When you delete Form submissions, Mautic may not always remove the corresponding records in the Form results table. To clean up these orphan records:
```

.. code-block:: shell

   bin/console mautic:forms:delete-orphan-form-submission-records-from-form-results-table

This command scans all Form results tables and removes records where the associated Form submission no longer exists. It processes records in batches of 5,000 to prevent performance issues on large databases.

.. note::

>    These commands are intended for one-time cleanup or periodic maintenance. Starting from the version where these commands were introduced, Mautic automatically deletes Form results table records when submissions are deleted through the UI.

Reviewer: Use active instead of passive voice.

```suggestion
   Mautic intends these commands for one-time cleanup or periodic maintenance. Starting from the version that introduced these commands, Mautic automatically deletes Form results table records when you delete submissions through the UI.
```
